src/Auth.ts

```diff
@@ -456,16 +456,7 @@ export class Auth {
       await logger.logToStderr('');
     }
 
-    cli.spinner.text = response.message;
-    cli.spinner.spinner = {
-      frames: ['🌶️ ']
-    };
-
-    // don't show spinner if running tests
-    /* c8 ignore next 3 */
-    if (!cli.spinner.isSpinning && typeof global.it === 'undefined') {
-      cli.spinner.start();
-    }
+    await logger.logToStderr(`🌶️  ${response.message}`);
```

Comment:
Questioning a bit why we always have two spaces between the🌶️ and our message. I noticed this was from the start already but never quite understood why 😄